test/io/write_sidb_layout_to_svg_in_siqad_style.cpp

```diff
@@ -0,0 +1,466 @@
+//
+// Created by Jan Drewniok on 05.11.24.
+//
+
+#include <catch2/catch_template_test_macros.hpp>
+#include <catch2/catch_test_macros.hpp>
+
+#include "utils/version_info.hpp"
+
+#include <fiction/io/write_sidb_layout_to_svg_in_siqad_style.hpp>
+#include <fiction/technology/cell_technologies.hpp>
+#include <fiction/technology/charge_distribution_surface.hpp>
+#include <fiction/technology/sidb_charge_state.hpp>
+#include <fiction/types.hpp>
+
+#include <fmt/format.h>
+
+#include <algorithm>
+#include <cctype>
+#include <sstream>
+#include <string>
+
+using namespace fiction;
+
+/**
+ * This function takes an SVG string as input and returns a new string with all
+ * whitespace characters removed. The purpose is to create a compact representation
+ * of the SVG content, which can be useful for comparisons, optimizations, or transmission.
+ *
+ * @param svg The input SVG string to be normalized.
+ * @return A string with all whitespace characters removed.
+ */
+auto normalize_svg = [](const std::string& svg) noexcept
```

Comment:
Objects in global namespace should at least be `const`, even better also `static`, even better yet `constexpr`. But also, why is this a lambda instead of a function?